### PR TITLE
Libretro: add pause button

### PR DESCRIPTION
**Pull request by SimpleTease**

### CODE REVIEW

File: common.cpp [hunk 1]
```diff
@@ -1051,9 +1051,38 @@ void mrboom_update_input(int keyid, int playerNumber, int state, bool isIA)
...
+         }
+      }
+   }
+   else if (!startPressed && keybounce)
    {
-      pressESC();
+      keybounce = false;
    }
 #endif
 }
```
Comment (frranck) [inline common.cpp:1088]: I would create an helper, isGameActiveInSolo() bool, but then what will happen if one player is looking at another playing over the network? Also NO_NETWORK has nothing to do with the  netplay, it's for the stats.

Comment (SimpleTease) [reply common.cpp:1088]: > but then what will happen if one player is looking at another playing over the network?

I have no answer.

Comment (frranck) [reply common.cpp:1088]: yeah it's very tricky... we should play a network game sometime though.

**Comment by frranck** [comment]
btw https://git.libretro.com/libretro/mrboom-libretro/-/pipelines/5429

**Comment by SimpleTease** [comment]
> btw https://git.libretro.com/libretro/mrboom-libretro/-/pipelines/5429

All platforms build for me so I'm guessing it's buildbot, but I have no clue why they break.

> we should play a network game sometime though.

You'd be disappointed at the domination. I lose 70% rounds. It gets worse against x bots.

**Comment by frranck** [comment]
any idea @twinaphex ?

**Pull request closed without merging**